aviatrix/resource_aviatrix_spoke_gateway.go

```diff
@@ -1420,6 +1430,7 @@ func resourceAviatrixSpokeGatewayRead(d *schema.ResourceData, meta interface{})
 	d.Set("enable_preserve_as_path", gw.EnablePreserveAsPath)
 	d.Set("rx_queue_size", gw.RxQueueSize)
 	d.Set("public_ip", gw.PublicIP)
+	d.Set("enable_global_vpc", gw.EnableGlobalVpc)
```

Comment:
"enable_global_vpc" does not have a default value. It may cause delta here since it is set for other cloud types as well.